tagpr.go

```diff
@@ -596,3 +574,32 @@ func (tp *tagpr) searchIssues(ctx context.Context, query string) ([]*github.Issu
 	}
 	return issues.Issues, nil
 }
+
+func buildChunkSearchIssuesQuery(queryBase string, shasStr string) (chunkQueries []string) {
+	query := queryBase
+	// Make bulk requests with multiple SHAs of the maximum possible length.
+	// If multiple SHAs are specified, the issue search API will treat it like an OR search,
+	// and all the pull requests will be searched.u
+	// This is difficult to read from the current documentation, but that is the current
+	// behavior and GitHub support has responded that this is the spec.
+	for _, sha := range strings.Split(shasStr, "\n") {
+		if strings.TrimSpace(sha) == "" {
+			continue
+		}
+		// Longer than 256 characters are not supported in the query.
+		// ref. https://docs.github.com/en/rest/reference/search#limitations-on-query-length
+		if len(query)+1+len(sha) >= 256 {
+			chunkQueries = append(chunkQueries, query)
+			query = queryBase
+
+			continue
```

Comment:
You shouldn't continue here. Current sha is should be appended to the query.

Reply:
Thank you.
I fixed it https://github.com/Songmu/tagpr/pull/121/commits/2f52564b0aa4ff697322495487211487c7e95f53